docs/UserGuide.md

```diff
@@ -4,9 +4,13 @@ title: User Guide
 ---
 
 TAlent Assistant™ is a **desktop, lightweight and centralized management system** catered to professors for managing
-the process of hiring undergraduate/graduate Teaching Assistants (TA). They will be able to access the candidates’ application
-data easily and review qualifications or availability for scheduling interviews. It is **optimized for use via a
-Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). If you can type fast, this application will be able to help you manage all things under the hood of the TA initiative faster than traditional GUI applications.
+the interview scheduling process of candidates applying to be undergraduate/graduate Teaching Assistants (TA). 
```

Comment:
Rationale of change - to improve phrasing and emphasise on core functionality/value proposition being the scheduling functionality. Qualifications of candidates is out of scope.

Adjusted phrasing to make subject of sentence clearer.

Reply:
Agreed!

Reply:
Yep, I agree too!